portfolio/src/main/webapp/script.js

```diff
@@ -27,5 +27,8 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-
-
+function getHelloArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('hello-container').innerHTML = quote;
+  });
```

Comment:
nit: A better way of writing this might be:
```
fetch('/data')
    .then(response => response.text())
    .then((quote) => {
        document.getElementById('hello-container').innerHTML = quote;
     });
```

Reply:
solved